coverage/src/BamViewChart.js

```diff
@@ -700,7 +700,7 @@ function createBamView(bamHeader, data, element, bamViewControlsElement, broker)
             .attr('dy', '.35em')
             .attr('text-anchor', 'start')
             .attr('fill', 'black')
-            .text(`Chr ${orgChromosome}: 0 - ${chromosomeEnd} (${chromosomeEnd} bp)`)
+            .text(`chr${orgChromosome}:1-${chromosomeEnd} (${chromosomeEnd} bp)`)
```

Comment:
This is going to result in invalid regions if the file doesn't use `chr` in front of the chromosome numbers.

Reply:
Sorry, I am unclear about "the file doesn't use chr". Can you explain to me with an example?

Reply:
Some files use "1, 2, 3, ..." instead of "chr1, chr2, chr3..." for the chromosome names (ie reference names). We need to detect and handle both cases.

Reply:
are you saying that the format could be `chr1:1-1000` or `1:1-1000`? 
<img width="772" alt="Screenshot 2024-08-21 at 4 59 04 PM" src="https://github.com/user-attachments/assets/2b579dbe-9ffa-4dd4-a089-4c763225fb3a">
So here we need to make the chromosome name align with the "SN" in the header file?

Reply:
Yep exactly. Match whatever the header uses.